apps/src/netsim/NetSimLogPanel.js

```diff
@@ -21,12 +21,20 @@ var NetSimEncodingControl = require('./NetSimEncodingControl');
 /**
  * Generator and controller for message log.
  * @param {jQuery} rootDiv
- * @param {string} logTitle
- * @param {boolean} [isMinimized] defaults to FALSE
+ * @param {Object} options
+ * @param {string} options.logTitle
+ * @param {boolean} [options.isMinimized] defaults to FALSE
+ * @param {packetHeaderSpec} options.packetSpec
  * @constructor
  * @augments NetSimPanel
  */
-var NetSimLogPanel = module.exports = function (rootDiv, logTitle, isMinimized) {
+var NetSimLogPanel = module.exports = function (rootDiv, options) {
```

Comment:
Just because I hit four arguments, and three is usually my limit.  We seem to have various points of view on this across the team.